packages/api-logs/index.jsx

```diff
@@ -88,7 +88,7 @@ class Logs extends React.Component {
   }
 
   onSelect(event) {
-    this.changeGroup(event.target.value);
+    this.onGroupChange(event.target.value, event.target.options[event.target.selectedIndex].text);
```

Comment:
Since `getAuth` asks for a `selectedApp` param, it makes more sense to push back the group name as well instead of only the group ID/api key to the explorer.